src/app/content/highlights/components/utils/showConfirmation.tsx

```diff
@@ -16,8 +18,15 @@ export default async(services: AppServices & MiddlewareAPI) => {
   root.insertAdjacentElement('afterend', domNode);
 
   const userChoice: boolean = await new Promise((resolve) => {
+    const focusedHighlight = focused(services.getState());
     const confirm = () => resolve(true);
-    const deny = () => resolve(false);
+    const deny = () => {
+      resolve(false);
+      if (focusedHighlight) {
+        services.dispatch(focusHighlight(focusedHighlight));
+      }
+      services.dispatch(setAnnotationChangesPending(true));
```

Comment:
i don't think it should be necessary to set this anymore, it should still be true already. it used to be necessary because we were unfocusing/refocusing the highlight and it cleared the editing state